### DISCUSSION TITLE
Changing the fonts used

### DISCUSSION THREAD

**Discussion by algoexpt** [pacollins/calligraphy#9] [Q&A] [answered]
Let's say I want to use a font different from Source Sans 3.

I had to do the following:
1. Change font-body in data/styles.toml
2. Change the import statement in assets/scss/main.scss (optionally the @font-face definitions if cdn = false)
3. Optionally download the ttf files and put them in static/fonts if cdn = false

But then I am basically changing the files in the repository directly.
Is there  a better way to do this?

**Comment by pacollins** [comment, marked as answer]
So at a basic level you can overwrite files at the site level rather than the theme level by taking advantage of [lookup order](https://gohugo.io/templates/lookup-order/). Lets imagine we are in your site directory, which includes `themes/calligraphy/data/styles.toml` that you want to modify. You can create your own at the root directory called `data/styles.toml` and it will overwrite the theme file.

That being said, there isn't an easy way to change the fonts beyond that which is an oversight. I'll have to think on the best solution, but I have an idea.

**Reply by algoexpt** [reply]
Thanks!
